Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -80,8 +91,18 @@ namespace Babylon::Plugins
         if (HasMouse())
         {
             // Create a mouse input map on initialization when available to match web behavior
-            const uint32_t inputIndex{ GetPointerButtonInputIndex(0) };
-            GetOrCreateInputMap(DeviceType::Mouse, MOUSE_POINTER_ID, { inputIndex, POINTER_X_INPUT_INDEX, POINTER_Y_INPUT_INDEX });
+            GetOrCreateInputMap(DeviceType::Mouse, MOUSE_POINTER_ID,
+                {POINTER_X_INPUT_INDEX, POINTER_Y_INPUT_INDEX, POINTER_BUTTON_LEFT_CLICK,
+                    POINTER_BUTTON_MIDDLE_CLICK,
+                    POINTER_BUTTON_RIGHT_CLICK,
+                    POINTER_BUTTON_BACK_CLICK,
+                    POINTER_BUTTON_FORWARD_CLICK,
+                    POINTER_MOUSEWHEEL_X,
+                    POINTER_MOUSEWHEEL_Y,
+                    POINTER_MOUSEWHEEL_Z,
+                    POINTER_DELTA_HORIZONTAL,
+                    POINTER_DELTA_VERTICAL,
+                    POINTER_MOVE});
```

Comment:
```suggestion
            GetOrCreateInputMap(DeviceType::Mouse, MOUSE_POINTER_ID, {
                    POINTER_X_INPUT_INDEX,
                    POINTER_Y_INPUT_INDEX,
                    POINTER_BUTTON_LEFT_CLICK,
                    POINTER_BUTTON_MIDDLE_CLICK,
                    POINTER_BUTTON_RIGHT_CLICK,
                    POINTER_BUTTON_BACK_CLICK,
                    POINTER_BUTTON_FORWARD_CLICK,
                    POINTER_MOUSEWHEEL_X,
                    POINTER_MOUSEWHEEL_Y,
                    POINTER_MOUSEWHEEL_Z,
                    POINTER_DELTA_HORIZONTAL,
                    POINTER_DELTA_VERTICAL,
                    POINTER_MOVE });
```

Reply:
Cleaned up organization and remove previously added values